src/StoreApi/Utilities/ValidationUtils.php

```diff
@@ -0,0 +1,64 @@
+<?php
+namespace Automattic\WooCommerce\StoreApi\Utilities;
+
+use \Exception;
+use Automattic\WooCommerce\StoreApi\Exceptions\RouteException;
+
+/**
+ * ValidationUtils class.
+ * Helper class which validates and update customer info.
+ */
+class ValidationUtils {
+	/**
+	 * Get list of states for a country.
+	 *
+	 * @param string $country Country code.
+	 * @return array Array of state names indexed by state keys.
+	 */
+	public function get_states_for_country( $country ) {
```

Comment:
This doesn't appear to be used publicly, only within this class. Can you confirm and change to `protected`?

Reply:
It's now used in AbstractAddressSchema so we can move it inline in both functions or just leave it public?

Reply:
Leave it if it's easier public.